audb/core/publish.py

```diff
@@ -381,6 +384,16 @@ def publish(
     In this case you don't call :func:`audb.load_to`
     before running :func:`audb.publish`.
 
+    When canceling :func:`audb.publish`
+    during publication
+    you can restart it afterwards
+    to continue from the current state.
+    But you might need overwrite permissions
+    in addition to write permissions
+    on the backend then.
```

Comment:
```suggestion
    When canceling :func:`audb.publish`
    during publication
    you can restart it afterwards.
    It will continue from the current state,
    but you might need overwrite permissions
    in addition to write permissions
    on the backend.
```